src/request/index.ts

```diff
@@ -206,6 +206,13 @@ export function jsonRequest(
 		}),
 		headers: options.headers
 	})
+		.then(response => {
```

Comment:
This could be a littler shorter if we put the `response.ok` check into the beginning of the `parseJSON` function.